service/worker/batcher/workflow.go

```diff
@@ -98,9 +98,11 @@ type (
 		Namespace string
 		// To get the target workflows for processing
 		Query string
+		// Target workflows for processing
+		Executions []*commonpb.WorkflowExecution
 		// Reason for the operation
 		Reason string
-		// Supporting: signal,cancel,terminate
+		// Supporting: signal,cancel,terminate,delete
```

Comment:
sometimes I hope there's a type for workflow refresh as well :).

Reply:
We should. Let's track this somewhere.